examples/parser/README.rst

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
As this is a pure-RST file, it might safely be moved to `doc/sources/...`

Reply:
But that is not necessary, right? I think it would be better if the file remains where it is.